src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Backup.Tests/RecoveryServicesBackupTestBase.cs

```diff
@@ -39,7 +39,7 @@ public class RecoveryServicesBackupTestBase : TestBase, IDisposable
 
         public RecoveryServicesBackupClient BackupClient { get; private set; }
 
-        public RecoveryServicesBackupTestBase(MockContext context)
```

Comment:
This is not what I meant.  See the discussion of class and collection fixtures here: https://xunit.github.io/docs/shared-context.html